src/utils.ts

```diff
@@ -61,8 +61,8 @@ export function saveModelToFile(dvConfig: IDVConfig, file: string): void {
 }
 
 export function getLabelFromKey(key: string, dsName: string, dsType: string): string {
-	let searchStr: string = `${dsType}_${dsName}_`.toUpperCase();
-	if (key.toUpperCase().startsWith(searchStr)) {
+	let searchStr: string = `${dsType}_${dsName}_`;
+	if (key.toUpperCase().startsWith(searchStr.toUpperCase())) {
```

Comment:
what's the link between this modification and the "fixed command naming"?

Reply:
nothing. it was a minor side change.